Sources/Plasma/FeatureLib/pfPython/pyAudioControlGlue.cpp

```diff
@@ -429,6 +455,9 @@ PYTHON_START_METHODS_TABLE(ptAudioControl)
     PYTHON_METHOD_NOARGS(ptAudioControl, getPriorityCutoff, "Returns current sound priority"),
     PYTHON_METHOD(ptAudioControl, setPriorityCutoff, "Params: priority\nSets the sound priority"),
     PYTHON_METHOD(ptAudioControl, enableVoiceChat, "Params: state\nEnables or disables voice chat."),
+    PYTHON_METHOD(ptAudioControl, setCaptureDevice, "Sets the name for the capture device being used by the audio system"),
```

Comment:
This is confusingly worded.  It might be better to say "Sets the capture device by name [...]"